### PR TITLE
Unified error handling - All controllers

### PR DESCRIPTION
**Pull request by jonmattgray**
Finish the work of https://github.com/digicatapult/veritable-cloudagent/pull/36 for the rest of the controllers

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "compilerOptions": {
     "paths": {
       "@aries-framework/rest": ["./src"],
-      "@src": ["src"],
+      "@src": ["./src"],
```
Comment (jonmattgray) [inline tsconfig.json:10]: IDE was giving me the error 
```
Non-relative paths are not allowed when 'baseUrl' is not set. Did you forget a leading './'
```

**Review by Ellenn-A — APPROVED** [review]
lgtm ...but no idea about the `leading'./'` error in tsconfig

File: tsconfig.json (hunk 1 above)
Comment (msterle) [reply tsconfig.json:10]: I haven't encountered this error, but you might try adding to the tsconfig `"BaseUrl": "."`

**Review by msterle — APPROVED** [review]
lgtm

**Pull request merged**